client/src/components/common/card/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Card from '@material-ui/core/Card';
+import useStyles from './style';
+
+const SimpleCard = ({ classname = 'card_steper', content }) => {
+  const classes = useStyles();
+  return <Card className={classes[`${classname}`]}>{content}</Card>;
+}
+
+export default SimpleCard;
```

Comment:
don't forget to add **props validation** also 
```suggestion
SimpleCard.propTypes = {
 className: PropTypes.string,
content:  PropTypes.string,
 }
 SimpleCard.defaultProps = {
 className: 'card_steper ',
 content : ' '
 }
 
export default SimpleCard;
```